code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -131,12 +131,15 @@
 	. = new /list
 	if(air)
 		var/list/gases = air.gases
-		for(var/id in gases)
-			var/gas = gases[id]
-			var/gas_meta = gas[GAS_META]
-			var/gas_overlay = gas_meta[META_GAS_OVERLAY]
-			if(gas_overlay && gas[MOLES] > gas_meta[META_GAS_MOLES_VISIBLE])
-				. += gas_overlay
+		if(gases)
+			for(var/id in gases)
+				if(GLOB.nonreactive_gases[id])
```

Comment:
this is chill for now tho :+1:

Reply:
actually no, better idea - populate a list of gases with overlays on startup and then cross reference with that one instead of non-reactive. this one can be built trivially